frontend/src/components/LayoutItem.module.css

```diff
@@ -0,0 +1,37 @@
+.item {
+  cursor: pointer;
+  background-color: #c5ced2;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0px 20px 20px 0px;
+  flex-grow: 1;
+  padding: 0 15px;
+  text-decoration: none;
+  color: #000;
+}
+
+.item_top {
+  border-top-left-radius: 20px;
+}
+
+.item_text {
+  font-size: 24px;
+  text-wrap: wrap;
+  text-align: center;
+  text-transform: uppercase;
+  max-width: 150px;
```

Comment:
Should there be max width if the `.item` which contains this text already has padding and the menu container has fixed width (if it doesn't make it fixed)?